install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -572,3 +572,7 @@ data:
   # Configure blacklisting of local routes not owned by Cilium.
   blacklist-conflicting-routes: {{ .Values.blacklistConflictingRoutes | quote }}
 {{- end }}
+
+{{- if hasKey .Values "crdWaitTimeout" }}
+  crd-wait-timeout: {{ .Values.crdWaitTimeout | quote }}
+{{- end }}
```

Comment:
I think we are almost there. There was another PR with similar changes and I think this is the right steps to add new options into the config map:
https://github.com/cilium/cilium/pull/12185#discussion_r442391297

Reply:
I've done it more similar way to how `defaultBpfCtAnyMax` variable is handled, since it might contain a custom value, not a true/false bool. Please look if you are fine with that.